python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -130,19 +129,28 @@ def __init__(self, *args, **kwargs):
                                                                      "NO_DATA", "DETECTED_NEGATIVE"]))
```

Comment:
The list of mask planes should be configurable rather than hard-coded.  It just needs a `ListField` in the `Config`.